problem-based-packs/insecure-transport/js-express/using-http-server.yaml

```diff
@@ -0,0 +1,21 @@
+rules:
+- id: using-http-server
+  patterns:
+  - pattern-either:
+    - pattern: |
+        http.$FUNC(...);
+    - pattern: |
+        new http.$FUNC2(...);
+  message: |
+    Checks for any usage of http servers instead of https servers. Encourages the usage of https protocol instead of 
+    http, which does not have TLS and is therefore unencrypted. Using http can lead to man-in-the-middle attacks 
+    in which the attacker is able to read sensitive information.
+  metadata:
+    vulnerability: Insecure transport
```

Comment:
This is probably OWASP 2017 Top 10: A3 Sensitive Data Exposure.
Would you also be able to add in the CWE? You can just Google "CWE http" or something simlar.

Reply:
yes definitely!!